decision_making.md

```diff
@@ -0,0 +1,45 @@
+
+Depending on the governing body, decisions may be proposed by members of the decision-making body or by a broader group of stakeholders/contributors.
+
+**Informal consensus seeking.** Decision making starts with informal consensus seeking through discussion. The goal of this phase is to refine the proposal, consider alternatives, weigh trade-offs, and attempt to find informal consensus. The legitimacy of the consensus-seeking process is predicated on all stakeholders having their voices heard, so teams must be proactive in providing opportunities for all relevant stakeholders to provide input. If the decision-making body arrives at informal consensus, they may immediately move to document and enact the decision. This is the consensus-seeking phase.
+
+**Calling a vote.** When the discussion matures during the consensus-seeking phase, any member of the decision-making body can call the matter to a vote. When that member (the sponsor) calls the vote, they shall summarize the proposal in its current form for the entire decision-making body. After the proposal is seconded by another member of the decision-making body, members have seven days to vote. The governing body may consider longer voting periods as necessary for special circumstances, or _shorter periods only if all voting members are present_. The decision will be determined by a simple majority of non-blank votes for binary decisions (i.e., approving a proposal) and ranked choice for multi-class decisions (one among many, or several among many). The sponsor may update the proposal at any point during the voting period, in which case the voting period will be reset.
+
+**Voter participation and quorum.** The Jupyter decision making bodies often have broad scope. There are typically a subset of members who have the time, interest, and expertise to dive into the details of a limited scope issue, such as subproject specific issues. We believe that open, transparent, and time-bound voting with no quorum will enable these ad-hoc, informal subgroups to form spontaneously for these limited scope issues and work on a per-decision basis, in a manner that supports our goals of responsiveness and engagement.
+
+All members of a decision-making body are required to participate in at least 2/3 of formal votes of that decision-making body per calendar year (teams can decide how to account for the specifics of this in low-activity projects, etc.). Members that have not met the 2/3 vote participation threshold for a year will automatically be asked to step down at the end of that year. Those individuals remain eligible to rejoin the decision-making body in the future as they become available to participate at the required level. The quorum for all formal votes will be 50% and a "blank" option will always be included, with the "blank" option counting towards the quorum but not included in totals for calculating results.
```

Comment:
In this paragraph, there is a mention of 50% quorum when in the previous one it says no quorum. I think that is confusing, at a minimum.

Reply:
Thanks for pointing out the inconsistency, @damianavila! This is resolved now.